From 6a022b482be96d6a74114a55619f13c0ac2d9c3d Mon Sep 17 00:00:00 2001
From: Alejandro Abdelnur <tucu@cloudera.com>
Date: Thu, 4 Oct 2012 11:27:38 -0700
Subject: [PATCH 0546/1357] HADOOP-8881. FileBasedKeyStoresFactory initialization logging should be debug not info. (tucu)

git-svn-id: https://svn.apache.org/repos/asf/hadoop/common/branches/branch-2@1394166 13f79535-47bb-0310-9956-ffa450edef68
(cherry picked from commit 2d1bc6e2b6e497586c8cc40d38b06628fd87901a)

Conflicts:

	hadoop-common-project/hadoop-common/CHANGES.txt
---
 .../security/ssl/FileBasedKeyStoresFactory.java    |    4 ++--
 1 files changed, 2 insertions(+), 2 deletions(-)

diff --git a/hadoop-common-project/hadoop-common/src/main/java/org/apache/hadoop/security/ssl/FileBasedKeyStoresFactory.java b/hadoop-common-project/hadoop-common/src/main/java/org/apache/hadoop/security/ssl/FileBasedKeyStoresFactory.java
index 00dd202..fef1aee 100644
--- a/hadoop-common-project/hadoop-common/src/main/java/org/apache/hadoop/security/ssl/FileBasedKeyStoresFactory.java
+++ b/hadoop-common-project/hadoop-common/src/main/java/org/apache/hadoop/security/ssl/FileBasedKeyStoresFactory.java
@@ -159,7 +159,7 @@ public class FileBasedKeyStoresFactory implements KeyStoresFactory {
       } finally {
         is.close();
       }
-      LOG.info(mode.toString() + " Loaded KeyStore: " + keystoreLocation);
+      LOG.debug(mode.toString() + " Loaded KeyStore: " + keystoreLocation);
     } else {
       keystore.load(null, null);
     }
@@ -200,7 +200,7 @@ public class FileBasedKeyStoresFactory implements KeyStoresFactory {
                                                  truststorePassword,
                                                  truststoreReloadInterval);
     trustManager.init();
-    LOG.info(mode.toString() + " Loaded TrustStore: " + truststoreLocation);
+    LOG.debug(mode.toString() + " Loaded TrustStore: " + truststoreLocation);
 
     trustManagers = new TrustManager[]{trustManager};
   }
-- 
1.7.0.4

